eclcli/bare/v2/server.py

```diff
@@ -548,13 +548,20 @@ def get_parser(self, prog_name):
             metavar="<server>",
             help="Name or ID of server",
         )
+        parser.add_argument(
+            "--type",
+            metavar="<type>",
+            choices=["SOFT", "HARD"],
```

Comment:
Can you switch them because you write help message with this order?
```suggestion
            choices=["HARD", "SOFT"],
```